zookeeper-server/src/test/java/org/apache/zookeeper/ZooKeeperTest.java

```diff
@@ -685,4 +686,25 @@ public void testSyncCommand() throws Exception {
         runCommandExpect(cmd, expected);
     }
 
+    @Test
+    public void testWaitForConnection() throws Exception {
+        // get a wrong port number
+        int invalid_port = PortAssignment.unique();
+        long timeout = 3000L; // millisecond
+        String[] args1 = {"-server", "localhost:" + invalid_port, "-timeout",
+            Long.toString(timeout), "-waitforconnection", "ls", "/"};
+        long start_time = System.currentTimeMillis();
+        // try to connect to a non-existing server so as to wait until wait_timeout
```

Comment:
using CamelCase naming style(e.g.: `startTime`) in the UT

Reply:
Used CamelCase naming style. Thanks @maoling for pointing out